### PR TITLE
Feature/tag finals

### PR DESCRIPTION
**Pull request by chrisbc**
* adding the new 1346 gridded / background seismicity data as default for crustal TargetMFDs. 
 * rebased on master for the the BG seismicity fixes
 * cleanup the Crustal Inversion Configuraiton classes for readability

### CODE REVIEW

**Comment by chrisbc** [comment]
@voj - sorry I forgot to rebase on main lastest. I'll do that now

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_AbstractInversionRunner.java [hunk 1]
```diff
@@ -535,59 +540,52 @@ private void appendMfdRows(EvenlyDiscretizedFunc mfd, ArrayList<ArrayList<String
...
-[4, solutionMFD_rateWeighted, 7.45, 8605.0]
-[4, solutionMFD_rateWeighted, 7.55, 10163.0]
-[4, solutionMFD_rateWeighted, 7.65, 10703.0]
-[4, solutionMFD_rateWeighted, 7.75, 11386.0]
-[4, solutionMFD_rateWeighted, 7.85, 9986.0]
-[4, solutionMFD_rateWeighted, 7.95, 5470.0]
-[4, solutionMFD_rateWeighted, 8.05, 362.0]
-
-
+ * [4, solutionMFD_rateWeighted, 6.05, 0.035982715383508626] [4,
```
Comment (voj) [inline src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_AbstractInversionRunner.java:553]: what are these things? should we just delete them? the new formatting messed them up a little.

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_CrustalInversionConfiguration.java [hunk 2]
```diff
@@ -3,47 +3,31 @@
...
- * individual model on the NZSHM22 logic tree. Parameters can be fetched for a
- * given logic tree branch with the <code>forModel(...)</code> method.
+ * individual model on the NZSHM22 logic tree.
  * 
  * based on scratch.UCERF3.inversion.UCERF3InversionConfiguration
  * 
  * @author chrisbc
  *
  */
+/**
```
Comment (voj) [inline src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_CrustalInversionConfiguration.java:19]: I wonder who's the author if this file? :-D

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_CrustalInversionConfiguration.java [hunk 3]
```diff
@@ -373,214 +193,19 @@ public static NZSHM22_CrustalInversionConfiguration forModel(InversionModels mod
 				.setSlipRateConstraintWt_normalized(slipRateConstraintWt_normalized)
 				.setSlipRateConstraintWt_unnormalized(slipRateConstraintWt_unnormalized)
 				.setSlipRateWeightingType(slipRateWeighting)
-				
-				.setMfdEqualityConstraints(mfdEqualityConstraints)
-				.setMfdInequalityConstraints(mfdInequalityConstraints)
+
+				.setMfdEqualityConstraints(mfdEqualityConstraints).setMfdInequalityConstraints(mfdInequalityConstraints)
```
Comment (voj) [inline src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_CrustalInversionConfiguration.java:197]: did you put this on the same line on purpose or was it done automatically?

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_CrustalInversionConfiguration.java [hunk 4]
```diff
@@ -373,214 +193,19 @@ public static NZSHM22_CrustalInversionConfiguration forModel(InversionModels mod
...
-				.setMfdInequalityConstraints(mfdInequalityConstraints)
+
+				.setMfdEqualityConstraints(mfdEqualityConstraints).setMfdInequalityConstraints(mfdInequalityConstraints)
 
 				// Rate Minimization config
 				.setMinimizationConstraintWt(minimizationConstraintWt)
 				.setMinimumRuptureRateFraction(minimumRuptureRateFraction)
-				.setMinimumRuptureRateBasis(minimumRuptureRateBasis)
-				.setInitialRupModel(initialRupModel);
+				.setMinimumRuptureRateBasis(minimumRuptureRateBasis).setInitialRupModel(initialRupModel);
```
Comment (voj) [inline src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_CrustalInversionConfiguration.java:202]: same here

**Review by voj — APPROVED** [review]
Looks good, only minor comments

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_CrustalInversionConfiguration.java (hunk 4 above)
Comment (chrisbc) [reply src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_CrustalInversionConfiguration.java:202]: weird - I think it's the formatter at work ?

**Pull request merged**